promql/value.go

```diff
@@ -168,6 +168,23 @@ func (p HPoint) MarshalJSON() ([]byte, error) {
 	return json.Marshal([...]interface{}{float64(p.T) / 1000, h})
 }
 
+// histogramSize returns the number of samples representing the histogram.
```

Comment:
How about: "histogramSize returns the size of the HPoint compared to the size of an FPoint. Or in other words: This HPoint is n times larger than an FPoint, where n is the returned number."